frontend/src/e2e-playwright/pages/employee/vasu/vasu.ts

```diff
@@ -128,9 +128,17 @@ export class VasuEditPage extends VasuPageCommon {
   }
 
   async waitUntilSaved(): Promise<void> {
-    await waitUntilEqual(
-      () => this.#vasuContainer.getAttribute('data-status'),
-      'clean'
+    const status = await this.#vasuContainer.getAttribute('data-status')
+    await new Promise((resolve) =>
+      setTimeout(
+        () => {
+          void waitUntilEqual(
+            () => this.#vasuContainer.getAttribute('data-status'),
+            'clean'
+          ).then(resolve)
+        },
+        status === 'clean' ? 500 : 0
+      )
```

Comment:
Discussed off-band: Try first if waiting for `dirty` and then waiting for `clean` fixes this.